File: code.leetcode/shuzu/easy/e217.存在重复元素.py
class Solution(object):
    def containsDuplicate(self, nums):
        #建map
        m={}
        #遍历
        for v in nums:
            #存在，返回

            #不存在，添加
            if v in m:
                return True
            
            m[v]=True
        	
        return False
            #存在，返回

            #不存在，添加

File: code.leetcode/shuzu/easy/e217.存在重复元素.cpp
#include <unordered_map>
class Solution
{
public:
    bool containsDuplicate(vector<int> &nums)
    {
        // 建map
        unordered_map<int, bool> m = {};
        // 遍历
        for (auto v : nums)
        {
            // 存在，返回

            // 不存在，添加
            if (m[v]==true)
                {
                    return true;
                }
            m[v] = true;
        }
        return false;
        // 存在，返回

        // 不存在，添加
    }
};